source/jormungandr/jormungandr/street_network/kraken.py

```diff
@@ -86,8 +86,22 @@ def _direct_path(
         if should_invert_journey:
             pt_object_origin, pt_object_destination = pt_object_destination, pt_object_origin
 
+        new_request = copy.deepcopy(request)
```

Comment:
minor: "new" is often too vague
```suggestion
        direct_path_request = copy.deepcopy(request)
```